range.c

```diff
@@ -1880,13 +1983,40 @@ range_alloc(VALUE klass)
 
 /*
  *  call-seq:
- *     range.count                 -> int
- *     range.count(item)           -> int
- *     range.count { |obj| block } -> int
+ *    count -> integer
+ *    count(&method) -> integer
+ *    count(object) -> integer
+ *    count {|element| ... } -> integer
+ *
+ *  Returns the count of elements, based on an argument or block criterion, if given.
+ *
+ *  With no argument and no block given, returns the number of elements:
+ *
+ *    (1..4).count      # => 4
+ *    (1...4).count     # => 3
+ *    ('a'..'d').count  # => 4
+ *    ('a'...'d').count # => 3
+ *    (1..).count       # => Infinity
+ *    (..4).count       # => Infinity
+ *
+ *  With argument +method+, returns the number of elements for which
```

Comment:
As above, this section should be removed as it is the same as the block usage.

Reply:
Done.